dev/PushNotifications/PushNotificationsLongRunningTask.StartupTask/winmain.cpp

```diff
@@ -3,19 +3,20 @@
 
 #include "../PushNotifications-Constants.h"
 
-#include <PushNotificationsLRP_h.h>
+#include <NotificationsLongRunningProcess_h.h>
 #include <wil/resource.h>
 
 int WINAPI WinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, PSTR /*lpCmdLine*/, int /*nCmdShow*/)
 {
+    Sleep(20000);
```

Comment:
Remove. This is only for debugging right?

Reply:
Yes, to allow some time to connect to WinDbg.